charts/simple/templates/configmap.yaml

```diff
@@ -71,6 +71,15 @@ data:
 
         map_hash_bucket_size        128;
 
+        map $http_host $x_robots_tag_header {
+            "~{{ template "simple.domain" $ }}$" "noindex, nofollow, nosnippet, noarchive";
+            {{- range $index, $prefix := .Values.domainPrefixes }}
```

Comment:
simple chart does not have `domainPrefixes` concept

Reply:
Resolved via https://github.com/wunderio/simple-project-k8s/pull/12/commits/dca923c64ccbcefee9c22e64d7675b82e5d54b04